torch/package/_package_pickler.py

```diff
@@ -94,9 +95,12 @@ def save_global(self, obj, name=None):
 
 
 def create_pickler(data_buf, importer):
+    protocol = 3
```

Comment:
Oh, I didn't realize protocol 4 is supported since 3.4. In that case, we don't need versioning guards; pytorch itself only supports python 3.6+.

However, we should add a global flag that users can set for now so that they can disable protocol 4 if they are running into any weird behavior

Reply:
Will do, but are there any examples of global flags lol